core/src/main/java/org/apache/druid/java/util/emitter/service/ServiceLogEvent.java

```diff
@@ -0,0 +1,154 @@
+package org.apache.druid.java.util.emitter.service;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.common.base.Predicate;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import org.apache.druid.guice.annotations.PublicApi;
+import org.apache.druid.java.util.common.DateTimes;
+import org.apache.druid.java.util.emitter.core.Event;
+import org.apache.druid.java.util.emitter.core.EventMap;
+import org.joda.time.DateTime;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.TreeMap;
+
+@PublicApi
+public class ServiceLogEvent implements Event
```

Comment:
add javadoc of what this should be used for. Its not a metric, its not an alert. its just an event that you might want emitted somewhere